object-manipulation-1/src/set-value.js

```diff
@@ -0,0 +1,15 @@
+/* exported setValue */
+
+/*
+1. check if object, key, orr value are equal to undefined
+  a. If they are, return the condition
+2. Assign the value of (object at key = value) to a new variable called newObj
+return newObj
+*/
+function setValue(object, key, value) {
+  if ((object === undefined) || (key === undefined) || (value === undefined)) {
+    return;
+  }
+  var newObj = object[key] = value;
```

Comment:
There is no need for this variable, you just need to do the work of assigning the value to the key. There is also no return for this function.
![Screen Shot 2021-11-03 at 3 31 59 PM](https://user-images.githubusercontent.com/32880187/140209234-851c9af8-9c8a-4294-8a7b-9d27fcec4b0c.png)